### ISSUE TITLE
hotplug support

### ISSUE THREAD

**Issue by GottZ** [odensc/yubioath-krunner#5]
as of right now you have to restart yubioath-krunner each time you plug in a yubikey.

instead it should be state aware and fetch credentials as soon as a yubikey is plugged in.

**Comment by GottZ** [comment]
maybe udev? https://forum.yubico.com/viewtopic303e.html?f=23&t=1143
maybe something else?

**Comment by odensc** [comment]
Good idea, will look into this more. I'm a bit iffy about using udev because it'll introduce a lot of complexity. Maybe we could poll `ykman list` every few seconds and see if a new YubiKey was added, and if so refresh the credential list.

**Comment by GottZ** [comment]
as a matter of fact ykman is written in python. i wonder if there is a decent binding that already does this job out of the box for long running processes.
